scripts/bootstrap-plugin

```diff
@@ -21,8 +21,6 @@ if [ -d $pluginname ]; then
   err "There is already a plugin with that name. Cowardly refusing to overwrite it."
 fi
 
-buildConfigFilename=${pluginname}/build.config.js
-
 mvn archetype:generate -DarchetypeGroupId=org.graylog -DarchetypeArtifactId=graylog-plugin-archetype -DartifactId=$pluginname
 
-sed 's,\(../graylog2-server/graylog2-web-interface\),../\1/,' < ${buildConfigFilename}.sample > ${buildConfigFilename}
+echo ${pluginname}/ >> ${prefix}/.gitignore
```

Comment:
I would move the cloned repositories into `modules/` or something similar and ignore the whole directory in `.gitignore`. Otherwise there will be modified files in the project. Also, this would add the directory multiple times if a user runs this script more than once.